packages/sdk/src/index.ts

```diff
@@ -1 +1,32 @@
-// TODO: expose all sdk packages as modules here
+import { authenticate, authorizeClient, UsernamePasswordAuthParams } from '@epilot/auth';
+import { getClient as getEntityClient, Client as EntityClient, OpenAPIClientAxios } from '@epilot/entity-client';
+import { getClient as getPricingClient, Client as PricingClient } from '@epilot/pricing-client';
+import { getClient as getUserClient, Client as UserClient } from '@epilot/user-client';
```

Comment:
Btw I propose we simplify this across all our clients by exporting the initialised client as a default export:

```suggestion
import entityClient from '@epilot/entity-client';
import pricingClient from '@epilot/pricing-client';
import userClient from '@epilot/user-client';
```

Then you can also do simply:

```typescript
export class EpilotClient {
  entity: typeof entityClient;
  pricing: typeof pricingClient;
  user: typeof userClient;
```